session_0/word_processing.rb

```diff
@@ -18,7 +27,13 @@ def lower_case!(words)
 # words_with_prefix('apple', ['apple', 'ball', 'applesauce']) would
 # return the words 'apple' and 'applesauce'.
 def words_with_prefix(prefix, words)
-  raise NotImplementedError # TODO
+  prefix_match = []
```

Comment:
Can we use `select` here? https://www.rubyguides.com/2019/04/ruby-select-method/

Reply:
Yes..We can use select as :-
words.select { |word| word.start_with?(prefix) }